### PR TITLE
Feature(oidc): add oidc sync job、use dcc public configure

### PR DESCRIPTION
**Pull request by 15168440402**

### CODE REVIEW

File: src/Services/Masa.Auth.Service.Admin/Application/Sso/CommandHandler.cs [hunk 1]
```diff
@@ -28,8 +23,11 @@ public class CommandHandler
         _userClaimRepository = userClaimRepository;
         _customLoginRepository = customLoginRepository;
         _eventBus = eventBus;
+        _syncCache = syncCache;
     }
 
+
```
Comment (MayueCif) [inline src/Services/Masa.Auth.Service.Admin/Application/Sso/CommandHandler.cs:29]: remove blank line

File: src/Services/Masa.Auth.Service.Admin/Infrastructure/Extensions/SchedulerJobServiceCollectionExtensions.cs [hunk 2]
```diff
@@ -76,4 +79,47 @@ await schedulerClient.SchedulerJobService.AddAsync(new AddSchedulerJobRequest()
...
+    }
+
+    async static Task SafeExcuteAsync(Task task)
+    {
+        try
+        {
+            await task;
+        }
+        catch
+        {
```
Comment (MayueCif) [inline src/Services/Masa.Auth.Service.Admin/Infrastructure/Extensions/SchedulerJobServiceCollectionExtensions.cs:122]: add log

File: src/Web/Masa.Auth.Web.Admin.Server/Program.cs [hunk 3]
```diff
@@ -1,33 +1,33 @@
 // Copyright (c) MASA Stack All rights reserved.
 // Licensed under the Apache License. See LICENSE.txt in the project root for license information.
 
+using Masa.Utils.Security.Authentication.OpenIdConnect;
```
Comment (MayueCif) [inline src/Web/Masa.Auth.Web.Admin.Server/Program.cs:4]: global using

**Review by MayueCif — CHANGES_REQUESTED** [review]
@15168440402

File: src/Services/Masa.Auth.Service.Admin/Program.cs [hunk 4]
```diff
@@ -6,30 +6,26 @@
 builder.Services.AddAutoInject();
 builder.AddObservability();
 
-#if DEBUG
-builder.Services.AddDaprStarter(opt =>
+//#if DEBUG
```
Comment (doddgu) [inline src/Services/Masa.Auth.Service.Admin/Program.cs:9]: ?

File: src/Services/Masa.Auth.Service.Admin/Program.cs [hunk 5]
```diff
@@ -69,17 +65,13 @@
...
-    configurationBuilder.UseDcc();
-});
 builder.Services.AddDccClient();
-var configuration = builder.GetMasaConfiguration().ConfigurationApi.GetDefault();
-builder.Services.AddMasaRedisCache(configuration.GetSection("RedisConfig").Get<RedisConfigurationOptions>());
-builder.Services.AddPmClient(configuration.GetValue<string>("AppSettings:PmClient:Url"));
-builder.Services.AddSchedulerClient(configuration.GetValue<string>("AppSettings:SchedulerClient:Url"));
+var defaultConfiguration = builder.GetMasaConfiguration().ConfigurationApi.GetDefault();
+builder.Services.AddMasaRedisCache(defaultConfiguration.GetSection("RedisConfig").Get<RedisConfigurationOptions>());
+builder.Services.AddPmClient(publicConfiguration.GetValue<string>("$public.AppSettings:PmClient:Url"));
```
Comment (doddgu) [inline src/Services/Masa.Auth.Service.Admin/Program.cs:71]: Why both . and : ?

**Review by doddgu — CHANGES_REQUESTED** [review]

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**